ansible.cfg

```diff
@@ -16,5 +16,5 @@ become = True
 
 [ssh_connection]
 ssh_args = -o ControlMaster=auto -o ControlPersist=900s
-control_path = %(directory)s/%%C
+control_path = %(directory)s/%%h-%%r
```

Comment:
Hi @thoraxe, I think this is related to https://groups.google.com/forum/#!topic/ansible-devel/52y8i7R8thU it seems that the %C Option is not available in RHEL 7.1. What was the error?